### PR TITLE
Add CORS example

### PR DESCRIPTION
**Pull request by JamesNK**
Replaces https://github.com/grpc/grpc-dotnet/pull/772

### CODE REVIEW

File: examples/Spar/Server/ClientApp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (JunTaoLuo) [inline examples/Spar/Server/ClientApp/.gitignore]: Why is there a .gitignore in here? Can it be moved to the root one instead?

File: examples/Spar/Server/ClientApp/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,7564 @@
+{
```
Comment (JunTaoLuo) [inline examples/Spar/Server/ClientApp/package-lock.json]: Does this need to be checked in?

Comment (JamesNK) [reply examples/Spar/Server/ClientApp/package-lock.json]: Yes

File: examples/Spar/Server/ClientApp/.gitignore (hunk 1 above)
Comment (JamesNK) [reply examples/Spar/Server/ClientApp/.gitignore]: A nested gitignore is a standard part of the template. I think it is fine.

**Pull request merged**